### PR TITLE
Fix/icub model config and fixation to world

### PR DESCRIPTION
**Pull request by nunoguedelha**
This fixes 2 issues:
- `iCubGazeboV2_5 fixed (no hands)` not found by Gazebo
   => caused by `model.config` wrong extension
- the same model, once inserted in the simulation world, should stay fixed in the air (virtual pole), but instead falls on its feet.
  => due to wrong link definition in the sdf defining the model position in the world.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ cd build
 cmake -DROBOT_NAME=iCubGenova04 ..
 ```
 
-After this, append `<icub-gazebo-wholebody>/build/gazebo/models` to the `GAZEBO_MODEL_PATH` enviromental variable.
+After this, append `<icub-gazebo-wholebody>/build/gazebo` to the `GAZEBO_MODEL_PATH` enviromental variable.
```
Comment (diegoferigo) [inline README.md]: I'm pretty sure with `.../models` was working. Trusting you in this.

**Review by diegoferigo — CHANGES_REQUESTED** [review]
The templated file `.in` is actually needed. For using this repo, a simple clone is not anymore an option. If you want to use it without installing, you need at least [to configure the project with CMake](https://github.com/robotology-playground/icub-gazebo-wholebody/tree/feature/useGeneratedModels#usage), and then point your env variables to the `build` folder.

File: README.md (hunk 1 above)
Comment (nunoguedelha) [reply README.md]: For the record, this was due to an issue in the CMakeLists.txt. to be fixed in the next commit

**Comment by nunoguedelha** [comment]
_Replying to https://github.com/robotology-playground/icub-gazebo-wholebody/pull/15#pullrequestreview-73701480_
> The templated file .in is actually needed. For using this repo, a simple clone is not anymore an option. If you want to use it without installing, you need at least to configure the project with CMake, and then point your env variables to the build folder.

The configuration through cmake was not renaming the `model.config.in` template to `model.config`. This has been fixed in the commit https://github.com/robotology-playground/icub-gazebo-wholebody/pull/15/commits/3e6407d7e4f9a68a702d3ca241fd7c311ead4d6d.

**Comment by nunoguedelha** [comment]
@diegoferigo @gabrielenava can you please review this PR last changes? Thanks.

**Review by diegoferigo — APPROVED** [review]

**Comment by diegoferigo** [comment]
LGTM now

**Review by gabrielenava — APPROVED** [review]
Ok for me, I'm going to test it soon

**Pull request merged**